src/mount/writedata.cc

```diff
@@ -318,11 +407,11 @@ void* delayed_queue_worker(void*) {
 		Glock lock(gMutex);
 		auto it = delayedQueue.begin();
 		while (it != delayedQueue.end()) {
-			if (it->inodeData == NULL) {
+			if (it->chunkData == NULL) {
```

Comment:
Consider adding a constant instead of directly using NULL. Check for other occurrences of this in new changes

Reply:
Done. Also added another constant for the inodedata.